ietf/templates/doc/shepherd_writeup.txt

```diff
@@ -39,8 +39,9 @@ to answer all of them.
 
 ### Additional Reviews
 
-5. Does this document need review from other IETF working groups or external
-   organizations? Have those reviews occurred?
+5. Does this document closely relate to work in other IETF working groups or
```

Comment:
I don't read the old text as being about "foundational technologies", at least not exclusively so. Anyway, how about
```suggestion
5. Does this document closely relate to technologies from other IETF working groups or
```

Reply:
There is a difference between getting the foundational technologies right (i18n, ABNF, ...) and interacting with tech from other SDOs.

If I work on COSE, should I get a review from OASIS DSS?

I think related is not the right term here.

Reply:
Suggest another one, please?

Reply:
interact?
(I'm not sure what exactly the change from the old text is trying to achieve.  This was focused on actually requiring review, not on abstract relation [or interaction].)

Also, the "what did the YANG doctor say" part is lost.

Reply:
The original text didn't really make sense though, because it first asked whether reviews are needed, and then asked what the results were.